src/function/open-referral-function/tests/FamilyHubs.OpenReferral.UnitTests/WhenUsingHsdaApiService.cs

```diff
@@ -79,40 +85,73 @@ public async Task Then_GetServices_Returns_NoContent_When_QueryIsOK_But_NoServic
     public async Task Then_GetServicesById_Returns_CorrectData()
     {
         _mockHttpMessageHandler.StatusCode = HttpStatusCode.OK;
-        _mockHttpMessageHandler.Content = "{\"contents\":[{\"id\":\"ABC\"}]}";
+        _mockHttpMessageHandler.Content = "{\"contents\":[{\"id\":\"00000000-0000-0000-0000-000000000000\"}]}";
```

Comment:
Links to my comment about guids above. Same elswhere

Reply:
Replied: https://github.com/DFE-Digital/fh-services/pull/92#discussion_r1763475029